src/components/message/ErrorPaper.module.css

```diff
@@ -0,0 +1,19 @@
+.paperWrapper {
+  max-width: 330px;
+  color: rgba(black, 0.87);
+  background-color: #f9cad3;
+  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0, 0.14),
+    0px 1px 3px 0px rgba(0, 0, 0, 0.12);
+}
+
+.paper {
+  display: flex;
+  justify-content: flex-start;
+  padding: 12px;
+  align-items: center;
+  gap: 7px;
```

Comment:
should we add flex-wrap to this, to ensure that it wraps if the screen size is to small to render the icon and the text side by side?